demo/src/main/java/com/google/maps/android/utils/demo/KmlDemoActivity.java

```diff
@@ -42,7 +42,7 @@ public void startDemo () {
 
     private void retrieveFileFromResource() {
         try {
-            KmlLayer kmlLayer = new KmlLayer(mMap, R.raw.campus, getApplicationContext());
+            KmlLayer kmlLayer = new KmlLayer(mMap, R.raw.campus, this);
```

Comment:
Note that this is a breaking change for apps, as any apps passing `getApplicationContext()` would need to change to passing an Activity, as shown here in the demo app. 

@jpoehnelt Normally this would require a new major version release in traditional semantic versioning, but given that we're still pre v1.0 I don't know your feelings on this.